Classes/EventStore/Storage/EventStorageInterface.php

```diff
@@ -22,6 +22,7 @@
  */
 interface EventStorageInterface
 {
+    public function __construct(array $options);
```

Comment:
We usually avoid constructors in the interface (see https://github.com/neos/flow-development-collection/blob/master/TYPO3.Flow/Classes/TYPO3/Flow/Http/Component/ComponentInterface.php#L26-L34 for example).
I just tested it and it seems that PHP7+ allows them, but our proxy building breaks (signature not the same..). Did that work for you?